docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -0,0 +1 @@
+The _Circuit Breaker_ Mission demonstrates a generic pattern for reporting the failure of a service and then limiting access to the failed service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
```

Comment:
@edewit WDYT? Would this be OK as the intro paragraph. Please let me know if you want me to shorten/expand/correct it.